github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1092,6 +1092,125 @@ periodics:
+    org: kubevirt
+    repo: kubevirt
+  - base_ref: main
+    org: kubevirt
+    repo: project-infra
+  labels:
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-kubevirtci-quay-credential: "true"
+    preset-shared-images: "true"
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  name: periodic-kubevirt-performance-cluster-density-test
+  max_concurrency: 1
+  spec:
+    containers:
+    - command:
+      - /usr/local/bin/runner.sh
+      - /bin/sh
+      - -c
+      - |
+        # install yq
+        curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+        chmod +x ./yq && mv ./yq /usr/local/bin/yq
+
+        # get kubectl
+        curl -L "https://dl.k8s.io/release/v1.19.7/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl
+        chmod +x /usr/local/bin/kubectl
+
+        # get kubeconfig
+        source ../project-infra/hack/manage-secrets.sh
+        decrypt_secrets
+        extract_secret 'kubeconfigPerf' /kubeconfig
+
+        # login quay.io
+        cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
+
+        # run the test
+        make clean
+        make bazel-build
+        make cluster-sync
+        make cluster-deploy
+        start_timestamp=$(date -u +%Y-%m-%dT%TZ
+        sleep 30
+        _out/cmd/perfscale-load-generator/perfscale-load-generator \
+          -container-prefix $DOCKER_PREFIX \
+          -container-tag $DOCKER_TAG \
+          -v 6 \
+          -workload tools/perfscale-load-generator/examples/workload/kubevirt-density/kubevirt-density.yaml
+
+        # collect metrics      
+        kubectl -n monitoring port-forward service/prometheus-stack-kube-prom-prometheus ${PROMETHEUS_PORT} &> /dev/null &
+        sleep 30
+        stop_timestamp=$(date -u +%Y-%m-%dT%TZ)
+        cat <<EOF >perfscale-audit-cfg.json
+        {
+          "prometheusURL": "http://127.0.0.1:${PROMETHEUS_PORT}",
+          "startTime": "$start_timestamp",
+          "endTime": "$stop_timestamp"
+        }
+        EOF
+        _out/cmd/perfscale-audit/perfscale-audit -config-file perfscale-audit-cfg.json
+      env:
+      - name: TARGET
+        value: external
+      - name: KUBEVIRT_PROVIDER
+        value: external
+      - name: KUBEVIRT_STORAGE
+        value: hpp
+      - name: DOCKER_PREFIX
+        value: quay.io/kubevirtci
+      - name: DOCKER_TAG
+        value: aarch64_test
+      - name: KUBECONFIG
+        value: /kubeconfig
+      - name: PROMETHEUS_PORT
+        value: "9090"
+      - name: kubectl
+        value: /usr/local/bin/kubectl
+      - name: IMAGE_PULL_POLICY
+        value: Always
+      - name: GIT_ASKPASS
+        value: ../project-infra/hack/git-askpass.sh
+      image: quay.io/kubevirtci/bootstrap:v20210906-994b913
+      name: ""
+      resources:
+        requests:
+          memory: 29Gi
```

Comment:
Same as above, this job doesn't need to spin up a cluster, it should be fine with much less memory, maybe 8 or even 4Gi

Reply:
Done!